Makefile

```diff
@@ -0,0 +1,60 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the MIT License. See the LICENSE accompanying this file
+# for the specific language governing permissions and limitations under
+# the License.
+
+pkgname=amazon-ec2-net-utils
+version=1.5
```

Comment:
Better to be `$(git describe --dirty)` ?

Reply:
We certainly can do that. Probably worth documenting an actual release process for this project so we can generate releases consistently. My issue with using tags for this is that we may have tags based on "upstream" releases and packaging releases (e.g. just the spec's Release field gets updated). I'm not sure `git describe` is going to get things consistently right without hints.